DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/Shared/_DelegatesSideNavMenu.cshtml

```diff
@@ -19,8 +19,9 @@
     link-text="Delegate groups"
     is-current-page="Model == DelegatePage.DelegateGroups" />
 
-  <vc:side-menu-link-with-href
-    href="@Configuration["CurrentSystemBaseUrl"]/tracking/coursedelegates"
+  <vc:side-menu-link

```

Comment:
Super tiny thing - could we also link the Top courses page link to here (and add to JIRA)?